src/actions/get-withdrawal-data.ts

```diff
@@ -57,13 +72,7 @@ export async function getWithdrawalInfo(txid: string) {
     const requestId = printEventDeserialized.value["request-id"]
       .value as bigint;
 
-    const withdrawal = await fetch(`${env.EMILY_URL}/withdrawal/${requestId}`);
-
-    if (!withdrawal.ok) {
-      throw new Error(`Failed to fetch withdrawal data: ${withdrawal.status}`);
-    }
-    const withdrawalData = await withdrawal.json();
-    status = WithdrawalStatus[withdrawalData.status as WithdrawalStatus];
+    return await getEmilyWithdrawal(String(requestId));
```

Comment:
We may want to suppress the errors from this one: there may be a time window in which the node we ask for the tx return success but Emily is still missing the withdraw. Ideally we also want to avoid redirecting until this is successful.

Reply:
Oh it may already be the case:
 - We have initial data without the requestId, so we would not redirect
 - If this fails, we return an error so the caller doesn't get a requestId

I guess the only possible issue is if the initial call to populate initialData ends up in the scenario above (or in general some fetch fails), thus returning an error: what happens in `WithdrawalStatusPage` if `getWithdrawalInfo` throws?

Reply:
Application error: a server-side exception has occurred (see the server logs for more information).
Digest: 3691147983

😭

Reply:
so I thought about this, and this scenario can't happen because the request id is never returned until `getEmilyWithdrawal` returns it, basically if for example if I start a withdrawal and shut down emily api the request will keep returning pending or break and it will keep retrying, the case we need to handle here is if emily is down and the url is a request id in this case we could have a simple 500/404 page that retries fetching indefinitely